vcd/metadata.go

```diff
@@ -358,7 +359,7 @@ func checkIgnoredMetadataConflicts(d *schema.ResourceData, vcdClient *VCDClient,
 					Severity: severity,
 					Summary:  "Found a conflict between 'ignore_metadata_changes' and 'metadata_entry'",
 					Detail: fmt.Sprintf("There is an 'ignored_metadata' block: %s\n"+
-						"and there is a 'metadata_entry' with key '%s' and value '%s' in your Terraform configuration that matches the criteria, hence it will be ignored.\n"+
+						"and there is a 'metadata_entry' with key '%s' and value '%s' in your Terraform configuration or state that matches that criteria, hence it will be ignored.\n"+
```

Comment:
```suggestion
						"and there is a 'metadata_entry' with key '%s' and value '%s' in your Terraform configuration or state that matches such criteria, hence it will be ignored.\n"+
```